bedrock/firefox/templates/firefox/more.html

```diff
@@ -232,14 +232,6 @@ <h2>{{ ftl('learn-more-about-firefox', fallback='firefox-products-are') }}</h2>
         link_url=url('firefox.browsers.compare.edge')
       )}}
 
-      {{ card(
-        title=ftl('comparing-firefox-ie'),
-        ga_title='Compare IE',
-        image=resp_img('img/firefox/more/firefox-ie.jpg', optional_attributes={"loading": "lazy", "class": "mzp-c-card-image"}),
```

Comment:
The `'media/img/firefox/more/firefox-ie.jpg'` is still around but it seems it was only ever linked from here. Should it be removed too?

@reemhamz I'm gonna PR pngcrushed versions of some of the changed images here, so I can include the removal of this stray asset with it if it's OK.